yoonit-camera/src/main/java/ai/cyberlabs/yoonit/camera/analyzers/face/FaceAnalyzer.kt

```diff
@@ -233,7 +235,8 @@ class FaceAnalyzer(
      */
     private fun handleDrawFaceDetection(
         faceBitmap: Bitmap,
-        faceDetectionBox: RectF
+        faceDetectionBox: RectF,
+        facePoints: MutableList<PointF>
```

Comment:
What do you  think `faceLandmarks` (and not `facePoints`)?

Because: 
- In the`captureOptions` is `faceLandmarks`;
- In theory, the "face points" are landmarks that is the base to find the face elements: left eye, right eye, nose...;